pull.js

```diff
@@ -20,6 +20,27 @@ function values (ary) {
   }
 }
 
+/*
+Or equivalently in the arrow function style:
+
+const values = ary => {
+  let i = 0
+  return (abort, cb) => (i < ary.length) 
+    ? cb(null, ary[i]) 
+    : cb(true)
+}
+*/
+
+// Usage example
+const show = console.log
+  , str = values([1,2,3])
+
+show("Pulling data from the stream...")
```

Comment:
why rename `console.log`? I can expect the reader to already know what console.log does, so they can understand each line, but now I had to go figure out what show was. At first I thought it was a module.